### DISCUSSION TITLE
Error building examples with componentsjs 5.2.0

### DISCUSSION THREAD

**Discussion by zg009** [LinkedSoftwareDependencies/Components.js#100] [Q&A]
Hello, I am trying to build off the examples in https://github.com/LinkedSoftwareDependencies/Examples-Components.js/tree/master/documentation/home/quick_start, however they haven't been updated for a while. I'm building using plain javascript, however I am getting the error -> Error: Failed to get module element MyComponent from module test-components. 
I attempted rerouting within folders, and have instantiated an exportable class named MyComponent in the components/ folder, and I was unable to solve the stack trace.

Attached below are my package.json and generated components error file as .txt.
[componentsjs-error-state.txt](https://github.com/LinkedSoftwareDependencies/Components.js/files/9085805/componentsjs-error-state.txt)
[package.txt](https://github.com/LinkedSoftwareDependencies/Components.js/files/9085806/package.txt)

Thanks in advance

**Comment by ixuz** [comment]
Hi @zg009 !

See [this small example](https://github.com/ixuz/minimal-cjs-example) that I put together to show how it can be done with the latest `components.js` version.

I also opened a pull request to update the example you linked: https://github.com/LinkedSoftwareDependencies/Examples-Components.js/pull/1

Hope it helps! 🙃

**Reply by rubensworks** [reply]
Thanks for the help @ixuz. I even forgot those examples were there.

@zg009 Be aware that the other examples in that repo may be outdated as well.
